OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -835,7 +835,7 @@ void TracerThread::ProcessMmapEvent(const perf_event_header& header,
 
   // There was a call to mmap with PROT_EXEC, hence refresh the maps.
   // This should happen rarely.
-  auto event = std::make_unique<MapsPerfEvent>(MonotonicTimestampNs(), ReadMaps(pid_));
+  auto event = std::make_unique<MapsPerfEvent>(pid_, MonotonicTimestampNs(), ReadMaps(pid_));
```

Comment:
This is not about this pull request, but I just realized that maybe we should read the exact timestamp from the record in the ring buffer rather than using `MonotonicTimestampNs` here.

Reply:
ops - I kind of missed that. Yes, will do it in follow-up cl.